packages/lxd/test/cluster/delete.coffee

```diff
@@ -0,0 +1,110 @@
+
+nikita = require '@nikitajs/core/lib'
+{tags, config} = require '../test'
+they = require('mocha-they')(config)
+path = require('path')
+
+return unless tags.lxd
+
+describe 'lxc.cluster.delete', ->
+
+  they 'delete a cluster', ({ssh}) ->
+    @timeout -1 # yum install take a lot of time
+    nikita
+      $ssh: ssh
+    , ->
+      cluster =
+        networks:
+          nktlxdpub:
+            'ipv4.address': '192.0.2.1/28'
+            'ipv4.nat': true
+            'ipv6.address': 'none'
+            'dns.domain': 'nikita.local'
+        containers:
+          'nikita-cluster-del-1':
+            image: 'images:centos/7'
+            nic:
+              eth0:
+                name: 'eth0', nictype: 'bridged', parent: 'nktlxdpub'
+          'nikita-cluster-del-2':
+            image: 'images:centos/7'
```

Comment:
Can you switch the image to alpine. The point for using centos/7 is because static IP and ssh are not supported for alpine yet, since we don't make use of those in this test, using alpine will speed them up.